lib/adapters/Memory.js

```diff
@@ -76,7 +76,7 @@ class Memory extends AbstractAdapter {
 							return true;
 						}
 					},
-					source: {
+					sourceMetadata: {
 						adapter: "Memory"
```

Comment:
might be worth to use here also a const variable for storing the adapter name `ADAPTER_NAME`

Reply:
Done